packages/xo-server/src/xo-mixins/vmware/index.mjs

```diff
@@ -38,8 +38,10 @@ export default class MigrateVm {
           ...OTHER_CONFIG_TEMPLATE,
           memory_dynamic_max: memory,
           memory_dynamic_min: memory,
-          memory_static_max: memory,
-          memory_static_min: memory,
+          // allow the user to increase the memory of this VM to the limit set by the template
+          memory_static_max: template.memory_static_max,
```

Comment:
Do not set the static max, usually all values but static min should all be equal.

Also, XO correctly updates it when changing the memory of a VM, the current issue concerns only static min :slightly_smiling_face: